bot/code_review_bot/report/base.py

```diff
@@ -121,7 +124,13 @@ def stats(analyzer, items):
         return [stats(analyzer, items) for analyzer, items in groups]
 
     def build_comment(
-        self, revision, issues, bug_report_url, patches=[], task_failures=[]
+        self,
+        revision,
+        issues,
+        bug_report_url,
+        patches=[],
+        task_failures=[],
+        frontend_url=None,
```

Comment:
Nit: Can we make this a required argument, so we can avoid checking it it is set